hdkeychain/extendedkey_test.go

```diff
@@ -1042,6 +1047,9 @@ func TestMaximumDepth(t *testing.T) {
 	}
 
 	for i := uint8(0); i < math.MaxUint8; i++ {
+		if extKey.Depth() != i {
+			t.Fatal("extendedkey depth %d should match expected value %d", extKey.Depth(), i)
```

Comment:
Please keep to 80 cols to be consistent with the rest of the code.

Also, it has to be `t.Fatalf` since it's a format string.